rca/editorial/models.py

```diff
@@ -0,0 +1,116 @@
+from django.db import models
+from modelcluster.fields import ParentalKey
+from wagtail.admin.edit_handlers import (
+    FieldPanel,
+    InlinePanel,
+    MultiFieldPanel,
+    PageChooserPanel,
+)
+from wagtail.api import APIField
+from wagtail.core.fields import RichTextField
+from wagtail.images.edit_handlers import ImageChooserPanel
+
+from rca.utils.models import BasePage, RelatedPage
+
+
+class EditorialPageRelatedSchoolsAndResearchPages(RelatedPage):
+    source_page = ParentalKey(
+        "EditorialPage", related_name="related_schools_and_research_pages"
+    )
+    panels = [
+        PageChooserPanel("page", ["schools.SchoolPage", "research.ResearchCentrePage"])
+    ]
+
+    api_fields = [APIField("page")]
+
+
+class Author(models.Model):
+    name = models.CharField(max_length=128)
+
+    def __str__(self):
+        return self.name
+
+
+class EditorialPageAuthor(models.Model):
+    page = ParentalKey("EditorialPage", related_name="editorial_authors")
+    author = models.ForeignKey("Author", related_name="+", on_delete=models.CASCADE)
```

Comment:
What does related_name="+" mean. I see it used q. a lot just not sure why the '+' is always used

Reply:
The + is a way of declaring the relationship won't work backwards.

> If you’d prefer Django not to create a backwards relation, set related_name to '+' or end it with '+'. For example, this will ensure that the User model won’t have a backwards relation to this model:

https://docs.djangoproject.com/en/2.2/ref/models/fields/#django.db.models.ForeignKey.related_name